pre-commit.sh

```diff
@@ -9,9 +9,15 @@ if ! [ -e venv ]; then
   python3 -m pip install virtualenv
   python3 -m virtualenv -p python3 venv
 fi
-venv/bin/pip install -r requirements-server.txt
-venv/bin/pip install -r requirements.docs.txt
-venv/bin/pip install -r requirements.dev.txt
+venv/bin/pip install -r requirements-server.txt --no-cache
+venv/bin/pip install -r requirements.docs.txt --no-cache
+venv/bin/pip install -r requirements.dev.txt --no-cache
+
+venv/bin/pip check || let val=$?
```

Comment:
you don't need the `||` or anything below since the script runs with `set -e`. When `pip check` exits with 1, the whole script will.